azurelinuxagent/common/conf.py

```diff
@@ -171,7 +171,8 @@ def load_conf_from_file(conf_file_path, conf=__conf__):
     "ResourceDisk.MountOptions": None,
     "ResourceDisk.Filesystem": "ext3",
     "AutoUpdate.GAFamily": "Prod",
-    "Policy.PolicyFilePath": "/etc/waagent_policy.json"
+    "Policy.PolicyFilePath": "/etc/waagent_policy.json",
+    "Protocol.EndpointDiscovery": "dhcp"
```

Comment:
Should I add this option to each config? Maybe commented out?

Reply:
I think the documentation in README.md is good enough